crypto/fipsmodule/ec/asm/p256-armv8-asm.pl

```diff
@@ -545,12 +521,12 @@
 	ret
 .size	__ecp_nistz256_sqr_mont,.-__ecp_nistz256_sqr_mont
 
-// Note that __ecp_nistz256_add expects both input vectors pre-loaded to
+// Note that __ecp_nistz256_add_to expects both input vectors pre-loaded to
 // $a0-$a3 and $t0-$t3. This is done because it's used in multiple
 // contexts, e.g. in multiplication by 2 and 3...
-.type	__ecp_nistz256_add,%function
+.type	__ecp_nistz256_add_to,%function
```

Comment:
Why renaming the function?

Reply:
I did that to avoid also a possibility of duplicate naming with ecp_nistz256_add in p256-nistz.c, because I recall that sometimes `__` is prepended to function names when compiling/linking. Another reason is that p256-x86_64-asm.pl has functions that have a similar name `add_tox`, `add_toq`.